.changeset/old-insects-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-aws-alb-provider': minor
+---
+
+**BREAKING** The `authModuleAwsAlbProvider` is now the default export and should be used like this in your backend: `backend.add(import('@backstage/plugin-auth-backend-module-aws-alb-provider'));`
```

Comment:
You could add the default export as a new thing and keep the old one (removing this warning). That way people on both the old and new can happily continue with the recommended patterns,